share/src/main/java/org/apache/kafka/server/share/persister/PersisterStateManager.java

```diff
@@ -679,9 +679,9 @@ protected void handleRequestResponse(ClientResponse response) {
                             // check retryable errors
                             case COORDINATOR_NOT_AVAILABLE:
                             case COORDINATOR_LOAD_IN_PROGRESS:
-                                log.warn("Received retryable error in read state RPC: {}", error.message());
+                                log.warn("Received retryable error in read state RPC for key {}: {}", partitionKey(), error.message());
```

Comment:
I know it's not quite a real word, but Kafka uses "retriable" not "retryable" such as in `RetriableException`. Please could you change the spelling here in the comments and log messages too to make searching easier?